src/response.cpp

```diff
@@ -71,6 +73,14 @@ operator=(
     return *this;
 }
 
+response::
+response(
+    http_proto::status sc)
```

Comment:
Hmm.. I don't know how I feel about this. Upon construction using this signature, the message will be invalid out of the box, because it is missing the required "Server" field.

Reply:
Good catch.

Reply:
Keep in mind, changing this from `version::http_1_1` to `version::http_1_0` means that this constructor allocates now, and a bunch of the tests seem predicated on this _not_ allocating so these will have to be adjusted.

Reply:
1_0 is not a good default